pennylane/tape/tape.py

```diff
@@ -525,7 +466,17 @@ def _update_circuit_info(self):
 
         self.is_sampled = any(m.return_type is Sample for m in self.measurements)
         self.all_sampled = all(m.return_type is Sample for m in self.measurements)
-        self.is_sampled = any(m.return_type is Sample for m in self.measurements)
```

Comment:
Oh was this just a random duplication, or something else?

Reply:
Accidental duplication that i noticed while going over the code.